locale/fr.po

```diff
@@ -796,25 +802,26 @@ msgstr "Impossible de lire la température"
 #: shared-bindings/_bleio/Adapter.c
 msgid "Cannot have scan responses for extended, connectable advertisements."
 msgstr ""
-"Impossible d'avoir des réponses d'analyse pour les publicités étendues et "
-"connectables."
+"Les publicités BLE étendues et connectables ne peuvent pas inclure de "
+"réponses au scan."
 
 #: ports/espressif/common-hal/alarm/pin/PinAlarm.c
 msgid "Cannot pull on input-only pin."
-msgstr "Ne peut tirer ('pull') sur une broche d'entrée ('input') seule."
+msgstr "Ne peut tirer ('pull') sur une broche d'entrée ('input') ."
 
 #: shared-bindings/audiobusio/PDMIn.c
 msgid "Cannot record to a file"
 msgstr "Impossible d'enregistrer vers un fichier"
 
 #: shared-module/storage/__init__.c
 msgid "Cannot remount '/' when visible via USB."
-msgstr "Ne peut démonter '/' quand est visible par USB."
+msgstr ""
+"Ne peut démonter '/' la racine du système de fichier quand elle est déja vue "
+"par le port USB."
 
 #: shared-bindings/digitalio/DigitalInOut.c
 msgid "Cannot set value when direction is input."
-msgstr ""
-"Impossible d'affecter une valeur quand la direction est entrante ('input')."
+msgstr "Impossible d'affecter une valeur quand le GPIO est entrant ('input')."
```

Comment:
The original message is more precise because it's talking about `direction`, which is a property.